includes/reports/data/payment-gateways/class-gateway-stats-list-table.php

```diff
@@ -97,35 +98,36 @@ public function get_data() {
 			$stats = new Stats();
 
 			$complete_count = $stats->get_gateway_sales( array(
-				'range'   => $filter['range'],
-				'gateway' => $gateway_id,
-				'status'  => array( 'complete', 'revoked' ),
-				'type'    => array( 'sale' ),
+				'range'    => $filter['range'],
+				'gateway'  => $gateway_id,
+				'status'   => array( 'complete', 'revoked' ),
+				'type'     => array( 'sale' ),
+				'currency' => $currency,
 			) );
 
-			//$complete_count = edd_count_sales_by_gateway( $gateway_id, 'publish' );
-
 			$pending_count = $stats->get_gateway_sales( array(
-				'range'   => $filter['range'],
-				'gateway' => $gateway_id,
-				'status'  => array( 'pending', 'failed' ),
-				'type'    => array( 'sale' ),
+				'range'    => $filter['range'],
+				'gateway'  => $gateway_id,
+				'status'   => array( 'pending', 'failed' ),
+				'type'     => array( 'sale' ),
+				'currency' => $currency,
 			) );
 
 			$refunded_count = $stats->get_gateway_sales( array(
-				'range'   => $filter['range'],
-				'gateway' => $gateway_id,
-				'status'  => array( 'complete' ),
-				'type'    => array( 'refund' ),
+				'range'    => $filter['range'],
+				'gateway'  => $gateway_id,
+				'status'   => array( 'complete' ),
+				'type'     => array( 'refund' ),
```

Comment:
Commenting here because I don't know where the heading is defined--but the heading on the table is "Refunded Sales" and I feel like that is not technically correct. I have two Euro payments, and one of them is partially refunded, but it's not being included in the `$complete_count` (which is maybe a different question), but this count is technically the number of _refunds issued_, not the number of sales which have been refunded. So if I have one order in a gateway, and then issue two separate partial refunds on the order, this column will show `2`, completed orders will show `0`, and the total sales will be `1`. It is a little sideways anyway but I'm basically wondering if we can change the heading to say just "Refunds Processed" or something like that?

Reply:
I think that's probably new issue territory. All I did in this class is add the currency logic. But what it's actually querying ('complete' refunds) hasn't changed. So I think we should open a new issue to look into that.

Don't milestone it for 3.0 though, as it's not a blocker to launch.